bot/.prettierrc

```diff
@@ -0,0 +1,18 @@
+{
```

Comment:
I think this would do just fine :)

```
{
  "singleQuote": true
}
```

Reply:
Updated `.prettierrc` to only have the options which are not defaults, it was more than just `"singleQuote": true`.